[Actions] Extend explanation for executeCommandline

We have to provide some extended information about the executeCommandline Action.
Especially an explanation on how to deal with multiple command arguments passed as different function parameters.
We should also add a small example with multiple parameters there.

Just typed this to create an issue in openhab-core before I discovered this issue.

executeCommandLine semantics have changed from OH2 to OH3:

**Regarding parameters**, see also (openhab/openhab-core#1700):

In OH2, `executeCommandLine("cat somefile")` worked as expected, actually running the command with a parameter.
In OH3, "cat somefile" is interpreted as a command, which is of course not found.
OH3 allows running `executeCommandLine("/bin/cat", "somefile")`, which finally works.

Error message when still using OH2 syntax is not very helpful, anyway:
```
[WARN ] [rg.openhab.core.io.net.exec.ExecUtil] - Failed to execute commandLine '[/bin/cat /var/log/openhab/callers.txt]'
[ERROR] [internal.handler.ScriptActionHandler] - Script execution of rule with UID 'callers-2' failed: cannot invoke method public java.lang.String[] java.lang.String.split(java.lang.String) on null in callers
```
Documentation does not describe this, see docs/configuration/actions.html#exec-actions.

**Regarding order of arguments** for the use case which includes waiting for timeout:
OH2: `executeCommandLine("/bin/cat somefile", 5000)`
OH3: `executeCommandLine(Duration.ofMillis(5000), "/bin/cat", "somefile")`

--- a/addons/actions.md
+++ b/addons/actions.md
@@ -62,8 +62,11 @@ You have different options to execute a command through an action.
 - `executeCommandLine(String commandLine)`: Executes a command on the command line without waiting for the command to complete
 For example you could run `executeCommandLine("path/to/my/script.sh")` which then would be executed and the rule would continue processing.
 
-- `executeCommandLine(Duration.fromSeconds(timeout), String commandLine)`: Executes a command on the command and waits `timeout` seconds for the command to complete, returning the output from the command as a String
-For example you could run `var ScriptResponse = executeCommandLine(Duration.fromSeconds(60), "path/to/my/script.sh");` would get executed and wait 1 minute for the output to be responded back and write it into the `ScriptResponse` variable.
+- `executeCommandLine(Duration.ofSeconds(timeout), String commandLine)`: Executes a command on the command and waits `timeout` seconds for the command to complete, returning the output from the command as a String
+For example you could run `var ScriptResponse = executeCommandLine(Duration.ofSeconds(60), "path/to/my/script.sh");` would get executed and wait 1 minute for the output to be responded back and write it into the `ScriptResponse` variable.
+
+Other Durations than `ofSeconds` units are possible too.
+Check out the [Java Documentation](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/Duration.html?is-external=true) for possible units.
 
 #### Scripts with parameters
 
@@ -82,7 +85,7 @@ The script above would be configured like shown below.
 executeCommandLine("path/to/my/script.sh", itemState1, itemState2);
 
 // When you need the output in your further rule processing
-var ScriptResponse = executeCommandLine(Duration.fromSeconds(60), "path/to/my/script.sh", itemState1, itemState2);
+var ScriptResponse = executeCommandLine(Duration.ofSeconds(60), "path/to/my/script.sh", itemState1, itemState2);
 ```
 
 ### HTTP Actions

--- a/addons/actions.md
+++ b/addons/actions.md
@@ -57,8 +57,33 @@ One can configure whether specific log entries are logged out and where they get
 
 ### Exec Actions
 
+You have different options to execute a command through an action.
+
 - `executeCommandLine(String commandLine)`: Executes a command on the command line without waiting for the command to complete
+For example you could run `executeCommandLine("path/to/my/script.sh")` which then would be executed and the rule would continue processing.
+
 - `executeCommandLine(Duration.fromSeconds(timeout), String commandLine)`: Executes a command on the command and waits `timeout` seconds for the command to complete, returning the output from the command as a String
+For example you could run `var ScriptResponse = executeCommandLine(Duration.fromSeconds(60), "path/to/my/script.sh");` would get executed and wait 1 minute for the output to be responded back and write it into the `ScriptResponse` variable.
+
+#### Scripts with parameters
+
+Let's assume that your `path/to/my/script.sh`-Script needs two item states to process them with some calculation.
+In console you would call it like
+
+```text
+path/to/my/script.sh itemState1 itemState2
+```
+
+To solve this constellation within a rule you have to add one argument per script parameter to the function.
+The script above would be configured like shown below.
+
+```text
+// When you are not interested in the script output
+executeCommandLine("path/to/my/script.sh", itemState1, itemState2);
+
+// When you need the output in your further rule processing
+var ScriptResponse = executeCommandLine(Duration.fromSeconds(60), "path/to/my/script.sh", itemState1, itemState2);
+```
 
 ### HTTP Actions
 

